x-pack/plugins/index_lifecycle_management/server/routes/api/nodes/register_list_route.ts

```diff
@@ -89,13 +89,16 @@ export function registerListRoute({
     license.guardApiRoute(async (context, request, response) => {
       try {
         const settingsResponse =
-          await context.core.elasticsearch.client.asCurrentUser.transport.request({
-            method: 'GET',
-            path: '/_nodes/settings',
-            querystring: {
-              format: 'json',
+          await context.core.elasticsearch.client.asCurrentUser.transport.request(
+            {
+              method: 'GET',
+              path: '/_nodes/settings',
+              querystring: {
+                format: 'json',
+              },
             },
-          });
+            { meta: true }
+          );
         const body: ListNodesRouteResponse = convertSettingsIntoLists(
           settingsResponse.body as Settings,
```

Comment:
Should we delete `body` property from this line? 
```suggestion
          settingsResponse as Settings,
```

Reply:
Nevermind, I see now that there is `meta: true` added to the request above.

Reply:
It's not a problem for this specific call, as I'm calling the client with `meta: true` just before:

https://github.com/elastic/kibana/blob/c756eae7b838bcb653502f6d50a0f1ed20b1b491/x-pack/plugins/index_lifecycle_management/server/routes/api/nodes/register_list_route.ts#L100-L101

But I'm not sure why I decided to use `meta: true` for this specific call. I can change it if you think it's better

Reply:
Done in https://github.com/elastic/kibana/pull/124488/commits/d1efec069786069da6e92d9047d7b1c5cb4dcbe0

Reply:
Thank you!